src/dotnet/Chat.UI.Blazor/Components/ChatPlaybackToggle.razor

```diff
@@ -25,13 +27,32 @@
         var isPlayingActive = await ChatUI.IsPlaying.Use(cancellationToken).ConfigureAwait(false);
         var chatPlaybackState = await ChatPlayers.ChatPlaybackState.Use(cancellationToken).ConfigureAwait(false);
         var isPlayingHistorical = chatPlaybackState is HistoricalChatPlaybackState;
+        if (!isPlayingActive)
+            return new Model() {
+                IsPlayingActive = isPlayingActive,
+                IsPlayingHistorical = isPlayingHistorical,
+            };
+        var listenedChatIds = await ChatUI.ListenedChatIds.Use(cancellationToken).ConfigureAwait(false);
+        ChatUI.ListenedChatIds.Value = AddToListened(listenedChatIds);
 
         return new Model() {
             IsPlayingActive = isPlayingActive,
             IsPlayingHistorical = isPlayingHistorical,
         };
     }
 
+    private ImmutableArray<Symbol> AddToListened(ImmutableArray<Symbol> lastListened) {
```

Comment:
`AddToListeningChatIds` + move it to `ChatUI`. We'll figure out how to do it better later.